islandora_transforms/RELS-EXT_to_solr.xslt

```diff
@@ -36,12 +35,7 @@
         <xsl:with-param name="value" select="@rdf:resource"/>
       </xsl:call-template>
     </xsl:template>
-
-    <!-- Match relevant literals, call underlying template.
-
-    We avoid indexing our compound "quad" relationship, which contains the PID appended after "isSequenceNumberOf".
-    -->
-    <xsl:template match="*[normalize-space(.)][self::islandora-rels-ext:isSequenceNumberOf or not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf'))]" mode="rels_ext_element">
```

Comment:
Should probably keep this, as it was [explicitly done in this fork](https://github.com/UCLALibrary/basic-solr-config/pull/8).